index.js

```diff
@@ -6,32 +6,59 @@ const { GitHub, context } = require('@actions/github')
 const sleep = dt => new Promise(resolve => setTimeout(resolve, dt))
 
 const main = async () => {
-  const token = core.getInput('github-token')
+  // GitHub token, also check underscore notation for console testing
+  const token = core.getInput('github-token') || core.getInput('github_token')
```

Comment:
Can you please explain when `github_token` would be supplied instead of `github-token`?

Reply:
Testing it from the console like this needs the variable name to not contain a hyphen:
INPUT_GITHUB_TOKEN=... GITHUB_REPOSITORY=Tantalon/spike-github-autorebase node index.js

Reply:
Ah, I see. I've never run this in my local console. Could we just rename the input then (so that it works for dev and prod), and make this a major release?

Reply:
Absolutely, I've committed this change.